unit-tests/language/fortran_test.py

```diff
@@ -218,3 +283,22 @@ def test_harvested_data(self, caplog, tmp_path):
         assert list(fdb.iterate_program_units()) \
             == [('barney_mod', [first_file, second_file]),
                 ('betty', [first_file])]
+        assert fdb.depends_on('betty') == ['barney_mod']
+
+    def test_naked_use(self, tmp_path):
+        '''
+        Ensures that an exception is raised if a "use" is found outside a
+        program unit.
+        '''
+        test_file: Path = tmp_path / 'test.f90'
+        test_file.write_text('''
+use beef_mod
+
+module test_mod
+end module test_mod
+''')
```

Comment:
Applies to other spots I expect, but the appearance of blocks like this can sometimes be made a bit neater looking by using the `textwrap` module, specifically `textwrap.dedent` which can remove all common whitespace indentation (allowing you to indent the sample Fortran code so that it looks nicer, but with the actual value being passed not having unnecessary leading whitespace)

Reply:
I was not aware of `textwrap.dedent()`, thanks.

Reply:
It's kind of a minor thing but I think it's nice for this... always seem to run into this kind of dilemma whenever you're dealing with a "language in a language" type situation (another obvious one is when you're using Jinja2 within another type of source; so difficult to know how to align/indent things so that it remains functional but doesn't look offensive)